freesound/static/bw-frontend/html/front.njk

```diff
@@ -191,6 +192,60 @@
             </div>
           </div>
         </section>
+        <hr />
+        <section class="v-spacing-2">
+          <h3 class="v-spacing-4 v-spacing-top-4">Trending packs</h3>
+          <div class="bw-carousel-container">
+            <div class="bw-carousel-wrapper">
+              <div class="bw-carousel v-padding-bottom-4">
+                <div>
+                  <div class="row no-margins">
+                    <div class="col">
+                      {{ players.smallPack('String music remixes', 5, 'john') }}
+                    </div>
+                    <div class="col">
+                      {{ players.smallPack('Ambient', 2, 'matthew__warrior') }}
+                    </div>
+                    <div class="col">
+                      {{ players.smallPack('Practicing guitar', 4, 'heavy_music') }}
+                    </div>
+                  </div>
+                </div>
+                <div>
+                  <div class="row no-margins">
+                    <div class="col">
+                      {{ players.smallPack('Cheerful', 3, 'happy') }}
+                    </div>
+                    <div class="col">
+                      {{ players.smallPack('Drone', 2, 'industrial') }}
+                    </div>
+                    <div class="col">
+                      {{ players.smallPack('Acoustic stuff', 4, 'indie4life') }}
+                    </div>
+                  </div>
+                </div>
+                <div>
+                  <div class="row no-margins">
+                    <div class="col">
+                      {{ players.smallPack('Drunk people at sonar', 5, 'giuband') }}
+                    </div>
+                    <div class="col">
+                      {{ players.smallPack('Noises of barceloneta', 7, 'giuband') }}
+                    </div>
+                    <div class="col">
+                      {{ players.smallPack('Traffic in Gran Via', 4, 'bcn_andy') }}
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+        <hr />
+        <section class="v-spacing-6">
+          <h3 class="v-spacing-4 v-spacing-top-4">Guess the sound of the day</h3>
```

Comment:
This is something that Marc proposed. The idea is that we display the random sound of the day without any metadata (only the player with waveform, no title). Then you can click somewhere (not implemented yet) to unravel the metadata. I think this is a nice idea to turn the sound of the day thing into a sort of game. Maybe "guess" is not good wording here?

Reply:
To me this seems a bit weird, but I'm not a designer, so...
Personally I like seeing the metadata on the front page before deciding if I want to listen to the sound

Reply:
this is a valid point as well. we can further discuss it. maybe once we have real full prototypes in place. code-wise this will be easy change anyway.